pkgs/development/ocaml-modules/type_conv/108.08.00.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ ocaml findlib ];
-  buildInputs = [ camlp4 ];
+  propagatedBuildInputs = [ camlp4 ];
```

Comment:
This does not fix the build failure with OCaml 4.02.

Reply:
Reverted all changes to `type_conv`